distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -15,20 +18,23 @@ function formatData(item) {
     result.push({
       x: data[i]['refRank'] || data[i]['b'],
       y: data[i]['c'] * 100,
+      p: data[i]['p'] * 100,
```

Comment:
I notice the numbers look a little strange. For example, the *Number of Bookmarks* chart goes from 93.2504% at the lowest up to 0.0004% at the highest. Are we using *p* correctly?